web/styles/compose.css

```diff
@@ -600,12 +599,8 @@ input.recipient_box {
 
 .open_enter_sends_dialog {
     font-size: 12px;
-    height: 14px;
-    padding-left: 4px;
+    height: 19px;
```

Comment:
Again, a variable or at least an explanatory comment would be useful for the `height` here...I see that you arrived at it by eliminating the 5px margin on the bottom, but we should document the origin of the 19px dimension, if possible.